pyafq_cli.json

```
@@ -0,0 +1,501 @@
+    "command-line": "pyAFQ [CONFIG_FILE]",
+    "schema-version": "0.5",
+    "container-image": {
+        "type": "docker",
+        "image": "gkiar/fuzzy:ubuntu14.04-python-dipy-afq"
+    },
+    "groups":[
+    {
+        "id": "bundles_group",
+        "name": "Bundles Group",
+        "description": "Parameters related to bundles of interest.",
+        "members": ["bundles", "bundles_seg_algo", "scalars_model", "scalars"]
+    },
+    {
+        "id": "tracking_group",
+        "name": "Tracking Group",
+        "description": "Parameters related to streamline tracking.",
+        "members": ["directions", "max_angle", "sphere", "seed_mask", "n_seeds",
+                    "random_seeds", "stop_mask", "stop_threshold", "step_size",
+                    "min_length", "max_length", "odf_model", "wm_labels"]
+    },
+    {
+        "id": "segmentation_group",
+        "name": "Segmentation Group",
+        "description": "Parameters related to segmentation.",
+        "members": ["nb_points", "segmentation_seg_algo", "progressive",
+                    "greater_than", "rm_small_clusters", "model_clust_thr",
+                    "reduction_thr", "refine", "pruning_thr", "b0_threshold",
+                    "prob_threshold", "rng", "segmentation_return_idx",
+                    "filter_by_endpoints", "dist_to_aal"]
+    },
+    {
+        "id": "cleaning_group",
+        "name": "Cleaning Group",
+        "description": "Parameters related to cleaning.",
+        "members": ["n_points", "clean_rounds", "distance_threshold",
+                    "length_threshold", "min_sl", "stat", "cleaning_return_idx"]
+    }
+    ],
+    "inputs": [
+    {
+        "id": "config_file",
+        "name": "Configuration file",
+        "type": "String",
+        "value-key": "[CONFIG_FILE]",
+        "optional": false
+    },
+    {
+        "id": "dmriprep_path",
+        "name": "dmriprep path",
+        "type": "String",
+        "value-key": "[DMRIPREP_PATH]",
+        "optional": false
+    },
+    {
+        "id": "bundles",
+        "name": "bundles",
+        "type": "String",
+        "list": true,
+        "min-list-entries": 1,
+        "list-separator": "', '",
+        "value-key": "[BUNDLES]",
+        "value-choices": ["ATR", "CGC", "CST", "HCC", "IFO", "ILF",
+                          "SLF", "ARC", "UNC", "FA", "FP"],
+        "default-value": ["ATR", "CGC", "CST", "HCC", "IFO", "ILF",
+                          "SLF", "ARC", "UNC", "FA", "FP"],
+        "optional": true
+    },
+    {
+        "id": "bundles_seg_algo",
+        "name": "bubndles seg algo",
+        "type": "String",
+        "value-key": "[BUNDLES_SEG_ALGO]",
+        "value-choices": ["AFQ"],
+        "default-value": "AFQ",
+        "optional": true
+    },
+    {
+        "id": "scalars_model",
+        "name": "scalars model",
+        "type": "String",
+        "value-key": "[SCALARS_MODEL]",
+        "value-choices": ["DTI"],
```

```suggestion
        "value-choices": ["DTI", "DKI"],
```

per:

>scalars_model is just "DTI" and "DKI" for now (and "DKI" is not really fully supported for now). In the long run, we'd like to also support inputs like ["DTI", "DKI"], where more than one model is fit, but we don't yet support that.

Eventually, we can turn this into a list so that the behaviour you'd like is possible